manim/mobject/coordinate_systems.py

```diff
@@ -202,7 +301,7 @@ def create_label_tex(label_tex) -> "Mobject":
             label_tex = MathTex(label_tex)
         return label_tex
 
-    def get_axis_label(
+    def _get_axis_label(
```

Comment:
Not sure whether this should be deprecated. I feel like I have seen people use this in code posted in our Discord, but I'll ultimately trust your judgement on this.

Reply:
... it might be that I'm thinking of `get_axis_labels`, however.

Reply:
I did a quick scan and I hadn't seen `get_axis_label` used. It's a bit messy to use when `get_x/y/z_axis_label` exist and are more convient, which is why I decided to make it private.